actions/call_api.py

```diff
@@ -0,0 +1,14 @@
+from lib.actions import ZabbixBaseAction
+from zabbix.api import ZabbixAPI
+
+
+class CallAPI(ZabbixBaseAction):
+    def run(self, api_method, token, **params):
+        # Initialize client object to connect Zabbix server
+        if token:
+            self.client = ZabbixAPI(url=self.config['zabbix']['url'])
+            self.auth = token
+        else:
+            self.connect()
+
+        return eval('self.client.%s' % api_method)(**params)
```

Comment:
Please do not use `eval`, Python has other ways to handle this.

```python
return eval('self.client.%s' % api_method)(**params)
```

can be replaced with:

```python
if not hasattr(self.client, api_method):
    raise Exception("Zabbix client does not have a '%s' method", api_method)
return getattr(self.client, api_method)(**params))
```